assets/src/edit-story/components/canvas/test/multiSelectionMovable.js

```diff
@@ -0,0 +1,139 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["rotateAndExpect"] }] */
+
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import Moveable from 'react-moveable';
+import PropTypes from 'prop-types';
+import React, { forwardRef } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { UnitsProvider } from '../../../../edit-story/units';
+import withOverlay from '../../overlay/withOverlay';
+import CanvasContext from '../context';
+import Selection from '../selection';
+import StoryContext from '../../../app/story/context';
+import TransformProvider from '../../transform/transformProvider';
+
+jest.mock('react-moveable', () => jest.fn(() => ({ children }) => children));
+
+const pageSize = { width: 100, height: 100 };
+
+const element1 = { id: '1', type: 'text' };
+const element2 = { id: '2', type: 'text' };
+
+const WrapperWithRef = ({ children }, ref) => (
+  <div ref={ref}>
+    <TransformProvider>
+      <UnitsProvider pageSize={pageSize}>{children}</UnitsProvider>
+    </TransformProvider>
+  </div>
+);
+const Wrapper = withOverlay(forwardRef(WrapperWithRef));
+
+WrapperWithRef.propTypes = {
+  children: PropTypes.node,
+};
+
+describe('multiSelectionMovable', () => {
+  let updateElementsById;
+  let updateSelectedElements;
+  let target1;
+  let target2;
+  let storyContext;
+  let canvasContext;
+
+  beforeEach(() => {
+    updateElementsById = jest.fn();
+    updateSelectedElements = jest.fn();
+    target1 = document.createElement('div');
+    target2 = document.createElement('div');
+
+    storyContext = {
+      state: { selectedElements: [element1, element2] },
+      actions: { updateElementsById, updateSelectedElements },
+    };
+    canvasContext = {
+      state: {
+        pageSize,
+        nodesById: {
+          '1': target1,
+          '2': target2,
+        },
+      },
+    };
+  });
+
+  function arrange() {
+    return render(
+      <StoryContext.Provider value={storyContext}>
+        <CanvasContext.Provider value={canvasContext}>
+          <Selection />
+        </CanvasContext.Provider>
+      </StoryContext.Provider>,
+      { wrapper: Wrapper }
+    );
+  }
+
+  function performRotatation(rotateTo) {
+    const moveable = Moveable.mock.calls[Moveable.mock.calls.length - 1][0];
+    moveable.onRotateGroupStart({ events: [{ set: () => {} }] });
+    moveable.onRotateGroup({
+      events: [
+        {
+          target: target1,
+          beforeRotate: rotateTo,
+          drag: { beforeTranslate: [0, 0] },
+        },
+        {
+          target: target2,
+          beforeRotate: rotateTo,
+          drag: { beforeTranslate: [0, 0] },
+        },
+      ],
+    });
+    moveable.onRotateGroupEnd({ targets: [target1, target2] });
+  }
+
+  describe('should rotate', () => {
```

Comment:
Nit: I'd name this 'Rotation'

Reply:
I think the test description should form a sentence, ie:
multiSelectionMovable › should rotate › "through 360 degrees"

This appears to be the convention used in other tests, for example:
https://github.com/google/web-stories-wp/blob/3a6511bbd62afeacb2a5e449f36b9f620ceb726e/assets/src/edit-story/app/story/useStoryReducer/test/setBackgroundElement.js#L133

Reply:
That‘s what I am also proposing, see comment below :-)
The „should rotate“ should be part of the „each“ cases though.

Reply:
I was thinking that the 'Rotation' part doesn't form a sentence, ie: 
"multiSelectionMovable > Rotation > should rotate %p"

but perhaps the 'rotation' group I added isn't needed. I've added a change to address this here:
https://github.com/google/web-stories-wp/pull/1710